### ISSUE TITLE
Hide mouse cursor in fullscreen

### ISSUE THREAD

**Issue by EVILEDLIBRE** [Windblade-GR01/Ikemen_GO#94]
Feature idea.

When I play the game full screen, I have to tuck the mouse courser in the bottom left or bottom right hand corner of the screen to hide it. Considering IKEMEN GO doesn't use the mouse courser at all in game, why this the mouse courser not made invisible in IKEMEN GO? Especially for full screen mode. But, if that would make it more difficult to program, why not disable it entirely? That way even if the courser rolled over it in window mode it would go invisible?

Even tucked away in the corner, I still see a little bit of it. It's visually distracting and does absolutely nothing in-game or even in menus. It would make much for sense for the program to just make it completely invisible.

**Comment by EVILEDLIBRE** [comment]
I just wanted to add another reason why this should be fixed. I have purchased a small TV to use for this game on its side and am thinking about making my own arcade cabinet with this game. They sell the framework on Etsy for about $300 and I might do it.

I have hooked up my custom made fight stick (SUZO HAPP) parts to my Mini PC and to my small TV to test all the mechanical and software elements to prep for my potential arcade cabinet build.

I installed Trisquel 9.0 Testing Version (I plan to use stable once stable releases) so as to be using only Free Software; Free as in Freedom.

I created a single user account and set it to not require my password for login; that way it would automatically log into the user account when the Mini PC is powered on.

I then went into startup programs and set IKEMEN GO to launch at startup in full screen at the proper resolution.

After some trial and error (realized the Binary had to launch from the home folder itself or it would not auto-launch from startup programs) I got almost everything working perfectly.

The mouse cursor, of course, is planted in the center of the game screen for no reason at all. The game doesn't use it for anything (in game-play or in menus.) When the Operating System starts; it assumes a centered mouse position.

So, if I do make my arcade cabinet, this would mean that every single time I power on my machine I will have to open up the door to get inside of my Mini PC to move the mouse cursor to the bottom right hand corner of the screen and then shut the door again. That's going to be awful. Even right now as an off the big TV on the small TV setup; I still have to reach away from my fight stick and move the mouse cursor every time to the bottom right to avoid this problem.

One temporary work around would be to find a way to make the cursor for the entire OS invisible. Maybe custom invisible cursor? Two problems with this, though. Every time I keep looking for how to make the cursor invisible on Google; I keep getting replies on how to fix an invisible cursor error. Not answers on how to achieve the effect on purpose. Maybe a cursor theme that's completely invisible?

Second, if I can figure out how to make the cursor invisible, this is going to make OS maintenance and addition of game elements a pain because I won't be able to see my cursor for adding new characters and stages in the GUI. (Open file browser and find text files) I'll just be clicking around until I can figure out where the cursor is so I can turn it on to find my way around the GUI to do the changes; only to have to turn it off again.

This would all be solved if the cursor just went invisible in fullscreen mode. Which, make sense, as it's not even being used. Many games do this. I'm not sure how hard it would be to do. But, it would save me from having to try to find a way to force the OS into making the cursor invisible for the entire OS.
